group_vars/gearshift_cluster/vars.yml

```diff
@@ -221,34 +221,34 @@ regular_users:
     sudoers: '%umcg-msb'
   - user: 'umcg-oncogenetics-dm'
     groups: ['umcg-oncogenetics']
-    sudoers: '%umcg-mterpstra'
+    sudoers: '%umcg-oncogenetics'
```

Comment:
missing suffix -dms?

Reply:
No, by design. For some groups we have a limited set of users that can become the DM. For other groups we allow all users of the group to become the DM. In this case there is only one user in that group anyway, so I skipped the overhead of creating a *-dms group.

Reply:
Ok!